### PR TITLE
Supernova

### PR DESCRIPTION
**Pull request by Lucia-Fonseca**
## Description
This is the `__init__.py` file for the supernovae working team. This addresses issue #154 
## Checklist
- [x] Follow the [Contributor Guidelines](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)
- [ ] Write unit tests
- [x] Write documentation strings
- [ ] Assign someone from your working team to review this pull request
- [x] Assign someone from the infrastructure team to review this pull request

### CODE REVIEW

File: skypy/supernovae/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+"""
+This module contains methods that model the intrinsic properties of supernovae
+Ia and core collapse contamination
```
Comment (rrjbca) [inline skypy/supernovae/__init__.py:3]: I think the module docstring you proposed is very specific - we are thinking about Ia cosmology and CC contamination now but we are likely to outgrow that scope in the future. Best to keep it general.

```suggestion
This module contains methods that model the intrinsic properties of supernovae.
```

Comment (Lucia-Fonseca) [reply skypy/supernovae/__init__.py:3]: That's right. Thanks

**Comment by Lucia-Fonseca** [comment]
> Also for consistency with other module names this should probably be singular `supernova`

Uhh you mean for the entire module. How do I change that now?

**Comment by philipp128** [comment]
> > Also for consistency with other module names this should probably be singular `supernova`
> 
> Uhh you mean for the entire module. How do I change that now?

Rename it with git move and push again if I remember correctly.

File: skypy/supernova/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+"""
+This module contains methods that model the intrinsic properties of supernova.
```
Comment (rrjbca) [inline skypy/supernova/__init__.py:2]: Typo (Ia)
```suggestion
This module contains methods that model the intrinsic properties of supernovae.
```

Comment (Lucia-Fonseca) [reply skypy/supernova/__init__.py:2]: Sure! I got confused when you said "supernova" singular

File: docs/supernova/index.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+*************************
+ Supernovae (`skypy.supernova`)
+ *************************
```
Comment (rrjbca) [inline docs/supernova/index.rst:3]: Formatting

```suggestion
******************************
Supernovae (`skypy.supernova`)
******************************
```

File: docs/index.rst [hunk 4]
```diff
@@ -4,7 +4,8 @@
 SkyPy Documentation
 ###################
 
-This package contains methods for modelling the Universe, galaxies and the Milky Way. Also included are methods for generating observed data.
+This package contains methods for modelling the Universe, galaxies and the
+Milky Way. Also included are methods for generating observed data.
```
Comment (rrjbca) [inline docs/index.rst:8]: This package contains methods for modelling the Universe, galaxies and the Milky Way. Also included are methods for generating observed data.

Comment (rrjbca) [inline docs/index.rst:8]:
```suggestion
This package contains methods for modelling the Universe, galaxies and the Milky Way. Also included are methods for generating observed data.
```

**Pull request merged**